changelog.mdx

```diff
@@ -5,6 +5,32 @@ metaDescription: Chalk's latest updates.
 description: Updates to Chalk!
 ---
 
+## January 6, 2025
+
+## Reduced cloud costs from optimized resource usage with KEDA
+
+We’re excited to share that we’ve implemented **KEDA (Kubernetes-based Event Driven Autoscaler)** and are rolling it out over the next few weeks.
+With KEDA, your environment now dynamically scales up and down—even to **zero machines** when there’s reduced or no workload.
+With automatic scaling we're able to optimize resource usage and reduce cloud costs--expect to see a drop in your cloud spend as a result.
+
+## View logs using the CLI
+
+View logs using the Chalk CLI! Inspect your log stream the same way you would from the GUI but via the command line!
+`chalk logs [--query="the same string you can put in the web ui"]`
+
+## Export and view metrics using the CLI
+
+View and export metrics using the Chalk CLI!
+In addition to exporting metrics from with a custom OpenMetrics compatible collector, you can now also export metrics directly using the CLI.
+`chalk metrics export`
+
+## Visualize and trace your feature's data lineage with a Graph View
+
+A new **graph view** in the web GUI--use the graph view to visualize the data lineage of your features and trace how your features are created and connected!
```

Comment:
i don't think this is proper use of the em-dash. if you have a sentence with one emdash i think it's better to make the preceding clause a complete clause.